scripts/master.sh

```diff
@@ -1,13 +1,22 @@
 #!/usr/bin/bash
 set -eu
 
+PRIVATE_IP=""
+while [ -z $PRIVATE_IP ]
+do
+  sleep 1
+  PRIVATE_IP=$(ifconfig | grep -A 1 'ens10' | tail -1 | awk '{print $2}'| cut -d ':' -f 2)
```

Comment:
Is that sure that the private will be placed on the `ens10` interface ? Also a retry mechanism would be good here. In case there is no private IP. Otherwise it will block the setup.